packages/polling-controller/src/StaticIntervalPollingController.test.ts

```diff
@@ -0,0 +1,242 @@
+import { ControllerMessenger } from '@metamask/base-controller';
+import { useFakeTimers } from 'sinon';
+
+import { advanceTime } from '../../../tests/helpers';
+import { StaticIntervalPollingController } from './StaticIntervalPollingController';
+
+const TICK_TIME = 1000;
+
+const createExecutePollMock = () => {
+  const executePollMock = jest.fn().mockImplementation(async () => {
+    return true;
+  });
+  return executePollMock;
+};
+
+class MyGasFeeController extends StaticIntervalPollingController<
```

Comment:
Does it matter that this is a gas fee controller? Perhaps `ChildStaticIntervalPollingController` would be a more suitable name.

Reply:
done here: https://github.com/MetaMask/core/pull/3636/commits/823aaa188f572a8aa76973a012ae9bfa8da518e0